torch/_dynamo/variables/nn_module.py

```diff
@@ -272,6 +272,16 @@ def record_nn_module_stack():
                     kwargs,
                 )
 
+    def _custom_getitem_fallback(self, module, tx, name, options, key):
+        getitem_fn = getattr(module, name).__func__
+
+        if not isinstance(getitem_fn, types.FunctionType):
+            unimplemented("torch.nn.Module with a non-function custom __getitem__")
+
+        return variables.UserMethodVariable(getitem_fn, self, **options).call_function(
+            tx, [key], {}
+        )
+
```

Comment:
Since this comes from ```call_method```, so we should assert ```__getitem__``` is always function and inline it.

Reply:
I see, will give this a try following https://github.com/pytorch/pytorch/blob/53c9bc8c68f05fee2421ef00c2f852f4cd322693/torch/_dynamo/variables/nn_module.py#L510